Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/SpectatorView.cs

```diff
@@ -23,6 +20,8 @@ public class SpectatorView : MonoBehaviour
         [SerializeField] MonoBehaviour RecordingServiceVisual;
         [SerializeField] List<MonoBehaviour> PlayerStateObservers;
 
+        [HideInInspector] public Matrix4x4 LocalOriginToSharedOrigin = Matrix4x4.identity;
```

Comment:
I suppose these aren't expected to be serialized as well?

I would make these properties

Reply:
Are properties not serialized in unity scenes?

Reply:
Nope, only fields, public or marked SerializeField

---
In reply to: [266633797](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3647#discussion_r266633797) [](ancestors = 266633797)